spec/system/candidate_interface/adviser_sign_up/candidate_selects_yes_on_adviser_interruption_spec.rb

```diff
@@ -0,0 +1,140 @@
+require 'rails_helper'
+
+RSpec.describe 'Candidate selects yes on adviser interruption' do
+  include CandidateHelper
+
+  it 'proceeds to adviser sign up flow' do
+    given_i_am_signed_in_with_one_login
+    and_adviser_teaching_subjects_exist
+    and_rails_cache_is_enabled
+    and_analytics_is_enabled
+    and_enqueued_jobs_are_not_performed
+    and_the_adviser_sign_up_feature_flag_is_enabled
+    and_the_get_into_teaching_api_is_accepting_sign_ups
+    and_adviser_sign_up_jobs_can_be_enqueued
```

Comment:
I'm not sure how important this text is to the narrative of the test. It might be better to put it in a before block to make the substance of the test easier to read.